compiler/resolution/functionResolution.cpp

```diff
@@ -8441,6 +8447,18 @@ static void handleRuntimeTypes()
   insertRuntimeInitTemps();
 }
 
+/*
+static void fixTypeNames() {
+  if (developer == false) {
+    forv_Vec(TypeSymbol, ts, gTypeSymbols) {
+      const char* typeName = toString(ts->type);
+      if (ts->name != typeName)
+        ts->name = typeName;
+    }
+  }
+}
+*/
+
```

Comment:
Were these left in accidentally?

Reply:
I wanted to show you what it could look like, if we went further in that direction.